pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -30,6 +30,45 @@
 import pandas.util.testing as tm
 
 
+def assert_invalid_comparison(left, right, box):
+    # Not for tznaive-tzaware comparison
```

Comment:
Should this be checked, raising if it's not the case?

Reply:
I don't think so.  If this is used incorrectly, this call will fail